web/send_error_test.go

```diff
@@ -35,16 +35,16 @@ func TestSendErrorSubProblemNamespace(t *testing.T) {
 		}),
 		"dfoop",
 	)
-	SendError(log.NewMock(), "namespace:test:", rw, &RequestEvent{}, prob, errors.New("it bad"))
+	SendError(log.NewMock(), rw, &RequestEvent{}, prob, errors.New("it bad"))
 
 	body := rw.Body.String()
 	test.AssertUnmarshaledEquals(t, body, `{
-		"type": "namespace:test:malformed",
+		"type": "urn:ietf:params:acme:error:malformed",
```

Comment:
What do you think about this so that if/when the ErrorNS ever changes, this is one less file to go edit?
```suggestion
		"type": `+probs.ErrorNS+`malformed",
```

Reply:
In general I'd totally agree! And that's exactly why we do what you suggest in e.g. wfe_test.go below. But this particular test function is here to ensure that the correct namespace is prefixed. So having it hard-coded here sorta acts as an extra check that the variable holds the correct value.